remotes/log.go

```diff
@@ -0,0 +1,15 @@
+package remotes
+
+import (
+	"encoding/json"
+
+	"github.com/sirupsen/logrus"
+)
+
+func logPayload(payload interface{}) {
+	buf, err := json.MarshalIndent(payload, "", "  ")
+	if err != nil {
+		return
+	}
+	logrus.Info(string(buf))
```

Comment:
Writing the payload at the `info` level feels a bit too verbose. Chatting offline, you said that containerd writes a lot of logs at the `debug` level. Maybe we should have two log levels – one for CNAB to OCI and one for containerd?

My guess is that the containerd logs should only be needed by tool developers using CNAB to OCI and that most people will only need the CNAB to OCI logs.

Reply:
Right, let me refactor that so the user can have a fine grained control on logs 👍